### ISSUE TITLE
Rendering for disused: and abandoned:landuse=quarry

### ISSUE THREAD

**Issue by matteocalosi** [gravitystorm/openstreetmap-carto#3535]
I know there has not been much love for the various proposals to render lifecycle prefix elements, in fact I don't think a single one has ever been rendered.

I do think disused/abandoned quarries at the very least deserve consideration for a series of reasons:
- Not as likely as other kinds of former landuse to have been reused as areas for another purpose, also no viable alternative tagging for the area if left unused, so if not rendered they'll remain empty spaces
- Very visible and useful landmarks, both on the ground and in satellite images
- If left untouched will be recognizable as former quarries for a very long time (centuries)
- Traditionally displayed on general use topo maps

The number is pretty small (about 700) but that's because there's more than 4000 uses of the deprecated disused and abandoned=yes tagging, supporting rendering might start fixing the ratio. I would also expect that a significantly larger amount of currently tagged quarries would fit the bill but have not been tagged as such for lack of local knowledge.

**Comment by kocio-pl** [comment]
Hi, thanks for the proposition, it sounds sane for me. Do you have some idea how should it look like? Sorry, I'm not familiar with this tagging and objects, are they well documented on the wiki or are there some unclear things?

**Comment by matteocalosi** [comment]
Wiki does state the current most used tagging scheme, I was under the impression that disused/abandoned=yes are deprecated and unused in rendering though?
I didn't really think about how to render maybe dashed borders? Or dashed/faded "hammers" symbol

**Comment by jeisenbe** [comment]
As the wiki states, the abandoned:x and disused:x are meant to prevent the
rendering of icons for things like shops, railways, roads and areas that
are not being used:

“Add the lifecycle prefix abandoned: to all keys which are no longer
relevant to the current state of an object. You should treat all the tags
on an object as a set of facts about the object, and prefix the keys of
those facts which are no longer true as a result of the abandonment. For
example, an abandoned shop which still carries a name sign might be tagged
as”

“abandoned:shop=doityourself
name=Holby Tool Supplies”

“which makes it unavailable to programs which locate shops, and unlikely to
be displayed as a shop in any default renderings. The name is still
relevant, and will still be searchable. If the shop is repaired and
reopened, it's a simple matter to undo the prefixing.”

So the question is if we should be rendering abandoned quarries.

Certainly the physical cliffs and embankments that make up the quarry can
be tagged and these will be rendered. The bare rock, sand, or scree can
also be tagged and will be rendered. Perhaps this is the best option.

Would it be more useful to render the quarry pattern and landuse color
instead, even for a quarry that is abandoned and will never be used?

**Comment by matteocalosi** [comment]
The current common tagging practice indicates that most mappers would prefer disused quarries to be rendered the same as active ones rather than not be displayed. 

Lifecycle prefix being born as a way not to render shops/amenities and the like makes it ambiguous when it comes to disused/abandoned prominent landmarks. 
Is landuse=quarry "relevant to the current state of the object" when no longer active? In some aspects it is (for example orientation or knowing that an area you're about to enter might be dangerous due to the presence of quarry cliffs), in other it isn't (anybody looking for active extraction sites). 
The first cases are probably the ones most people look for in a general use map, which leads to the use of deprecated disused=yes rather than the lifecycle prefix. Also rendering lifecycle prefixed quarries (abandoned and disused don't seem to be worth distinguishing), slightly differently than active ones, would fix the issue.
Railways already have their special redundant tagging scheme to get around the issue of still getting rendered as landmarks when disused (https://wiki.openstreetmap.org/wiki/Tag:railway%3Ddisused), more niche items never really got looked into (disused:aerialway is another case that should probably warrant rendering discussed on the Italian mailing list that sparked me to bring up this issue).

Of course it is possible to describe every single surface and embankment in disused quarries, it's extra information (much more time-consuming and difficult to map than the general perimeter), adding it does not make useless the rendering of the quarry area as a whole.

**Comment by jeisenbe** [comment]
> rendering lifecycle prefixed quarries...
> slightly differently than active ones, would fix the issue

Do you have a suggestion for how we can distinguish them? It’s tricky to
figure out a good rendering, especially for areas.

**Comment by matteocalosi** [comment]
I have no rendering experience so I'm unable to test alternatives or tell what would be the simplest choice.
Would changing the color of the crossed hammers symbol from white to light gray so it's a bit less visible against the gray background work?

**Comment by jragusa** [comment]
As I am geologist, I work in this old quarries because they provide nice outcrops. From my experience, there is 2 different evolution of old quarries.
1/ In the first group, old quarries remains as such and anyone would think they still active: the cliff remains clear, there is no vegetation on the ground or very few. Some of them are even reused to store inert materials (concrete, bricks, etc) locally or globally.
2/ The second ones looks really abandoned and people are sometimes not aware that there was a quarry here. Vegetation is crowding the site and the cliff is frequently hidden by the nascent forest.

The first group could be displayed with the current quarry pattern because it not so easy to define if the quarry is really disused or if some companies still use it to store materials or construction waste. They can also be tagged as `landuse=landfill`. Moreover this kind of site remains dangerous because the cliff is not controlled and can be easy of access. Finally, they are not necessarily closed by a fence.
The second one can be reorganised. Cliff can be tagged as `man_made=embankment` and the surface as natural=wood or landuse=forest or even natural=scrub. In some other case, the cliff collapse and form something like a scree. In other places, they have covered in variable degree a by large amount of earth collapsed from the top of the cliff. In these cases, `natural=scree` is preferable.

In conclusion, I am inclined to support `abandoned:landuse=quarry` and `disused:landuse=quarry`. It's up to the mapper to define if the site still recognisable or not (e.g. use other tags as above). Furthermore, some quarry name can remain known by local peoples.

**Comment by jeisenbe** [comment]
Stage 1 could be disused, stage 2 would be abandoned, no?

Perhaps we should only render disused quarries?

**Comment by jragusa** [comment]
Stage 1 could be disused or abandoned. It depends how is managed the site by the owner but this kind of information is not really use of access. Furthermore, an abandoned site stills recognisable at the beginning and the transformation to stage 2 can take some time before to be effective. However, yes, stage 2 is clearly abandoned.

**Comment by jragusa** [comment]
Please notice also that some abandoned quarries can also be transformed in archaeological sites. This the case with some large grind quarries for example.

**Comment by matkoniecz** [comment]
https://taginfo.openstreetmap.org/keys/disused%3Alanduse is used 1607 times, with 427 for `disused:landuse=quarry` ( https://taginfo.openstreetmap.org/tags/disused%3Alanduse=quarry )

I was not checking it but I guess that this tags are also not included in default presets of any popular editor.

**Comment by polarbearing** [comment]
IIRC the symbol in German maps is crossed hammers, head up for active ones, and hammer heads down for inactive ones.

**Comment by kocio-pl** [comment]
Smart idea!

**Comment by jragusa** [comment]
@matkoniecz I agree with you but I also suspect that the poor occurrence could be related to the lack of rendering. Moreover I don't think that all mappers investigate the activity of a quarry which is not necessarily continous. For example, in some case, the volume of some specific rocks is so low that the extraction is restricted to a low volume per year.

@polarbearing yes

**Comment by matthijsmelissen** [comment]
Oxford dictionaries defines a quarry as 'a place, typically a large, deep pit, from which stone or other materials are or have been extracted'. That would imply that inactive quarries are still quarries. I'm not sure if re-tagging abandoned quarries with something else than landuse=quarry would be a good idea.

**Comment by matthijsmelissen** [comment]
Oxford dictionaries defines a quarry as 'a place, typically a large, deep pit, from which stone or other materials are or have been extracted'. That would imply that inactive quarries are still quarries. I'm not sure if re-tagging abandoned quarries with something else than landuse=quarry would be a good idea.

**Comment by matkoniecz** [comment]
> I also suspect that the poor occurrence could be related to the lack of rendering.

That is why I also mentioned "I guess that this tags are also not included in default presets of any popular editor".

If this tag would be included in presets of all/some/any serious editor I would consider the possibility that is a correct tagging that was swamped by tagging for renderer.

**Comment by matkoniecz** [comment]
For what it is worth see https://wiki.openstreetmap.org/w/index.php?title=Tag:landuse%3Dquarry&oldid=1681654#Former_quarries (I will add note based on https://github.com/gravitystorm/openstreetmap-carto/issues/3535#issuecomment-443270218 )

> Quarries that are out of use but still carry many characteristics of a quarry shall be tagges as {{tag|landuse|quarry}} and  {{tag|1=disused|2=yes}}. 

> Quarries that are out of use for a longer time and not any more clearly recognized as a quarry shall not be mapped as {{tag|landuse|quarry}} but only with lifecycle prefix [[Key:abandoned:|abandoned:]] while the scarps may be mapped with {{tag|man_made|embankment}}. Maybe it's meaningful to use {{Tag|note}}, {{Tag|description}} or the like to transport further information, e.g. that this structure has been a quarry.

**Comment by jragusa** [comment]
Following comments of @matthijsmelissen and @matkoniecz , this issue raised for me an overall question about using prefix `abandoned:`, `disused:`. Is that a valid tagging scheme or not for polygons such as `landuse=*` ? Is there any discussion about it in the tagging list ?

This issue would have some side effects such as the rendering of `abandoned:landuse=forest` for example.

**Comment by matkoniecz** [comment]
> Is there any discussion about it in the tagging list ?

No idea.

> Is that a valid tagging scheme or not for polygons such as landuse=* ? 

I prefer to discuss it on tagging mailing list, but it may depend on landuse value. As discussed above it seems to not be any feature that can be tagged as `abandoned:landuse=quarry`
